rep-2000.rst

```diff
@@ -418,6 +418,112 @@ Build System Support:
+|    amd64     |      Tier 1 [d][a][s] |           Tier 1 [a][s]      |          Tier 1 [a][s] |        Tier 3 [s]      |
++--------------+-----------------------+------------------------------+------------------------+------------------------+
+|    arm64     |      Tier 1 [d][a][s] |                              |                        |        Tier 3 [s]      |
++--------------+-----------------------+------------------------------+------------------------+------------------------+
+|    arm32     |        Tier 2 [a][s]  |                              |                        |        Tier 3 [s]      |
++--------------+-----------------------+------------------------------+------------------------+------------------------+
+
+The following indicators show what delivery mechanisms are available for each platform.
+
+" [d] " Debian packages will be provided for this platform for packages submitted to the rosdistro.
+
+" [a] " Binary releases are provided as a single archive per platform containing all packages in the Eloquent ROS 2 repos file [6]_.
+
+" [s] " Compilation from source.
+
+Middleware Implementation Support:
+
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+| Middleware Library       | Middleware Provider | Support Level | Platforms                   | Architectures                        |
++==========================+=====================+===============+=============================+======================================+
+|  rmw_fastrtps_cpp        | eProsima Fast-RTPS  | Tier 1        | All Platforms               | All Architectures                    |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+|  rmw_connext_cpp         | RTI Connext         | Tier 1        | All Platforms except Debian | All Architectures except arm64/arm32 |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+| rmw_opensplice_cpp       | ADLink OpenSplice   | Tier 2        | All Platforms except Debian | All Architectures                    |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+| rmw_fastrtps_dynamic_cpp | eProsima Fast-RTPS  | Tier 2        | All Platforms               | All Architectures                    |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+
+Middleware implementation support is dependent upon the platform support tier.
+For example a Tier 1 middleware implementation on a Tier 2 platform can only receive Tier 2 support.
+
+Minumum language requirements:
+
+- C++14
+- Python 3.5
```

Comment:
This should be 3.6 as the lowest common denominator with Stretch replaced by Buster.